roles/ceph-osd/tasks/docker/start_docker_osd.yml

```diff
@@ -64,23 +71,15 @@
   become: true
   template:
     src: "{{ role_path }}/templates/ceph-osd.service.j2"
-    dest: /var/lib/ceph/ceph-osd@.service
+    dest: /etc/systemd/system/ceph-osd@.service
```

Comment:
This won't work for multiple OSDs.  That's why it was set up to copy once and symlink.  If you want to copy instead of symlink, you will need to copy once per OSD and use the naming scheme from the symlink task, I think.